loleaflet/src/control/Parts.js

```diff
@@ -279,7 +279,7 @@ L.Map.include({
 		var docLayer = this._docLayer;
 
 		// At least for Impress, we should not fire this. It causes a circular reference.
-		if (this.getDocType() !== 'presentation') {
+		if (this.isPresentationOrDrawing()) {
```

Comment:
Here you rewrote this incorrectly. It should be `if (!this.isPresentationOrDrawing()) {` I assume.

Reply:
@tzolnai: Thanks for that, @merttumer has fixed that in an additional commit.